### DISCUSSION TITLE
Webhook support

### DISCUSSION THREAD

**Discussion by bachelor-emgi** [RaspAP/raspap-webgui#1372] [Ideas]
Basically the webhook could send statistics about new devices connected to RaspAP hotspot by webhook let's say to discord.

It would be pretty nice to have some alerting like this for new connected devices or new updates for RaspAp so we can get notified by it too.

**Comment by bachelor-emgi** [comment]
For example Uptime Kuma has it integrated (see below images)

![Screenshot_20230801-131849_Chrome~2](https://github.com/RaspAP/raspap-webgui/assets/26904790/7bf481cf-d49d-4f6f-8d5f-c666a51c159e)
![Screenshot_20230801-132037_Discord](https://github.com/RaspAP/raspap-webgui/assets/26904790/25ee8eea-dddb-412d-9f08-5898ec078ab3)

**Comment by billz** [comment]
Nice idea. There have been some related discussions around this in the past #602 #393
A webhook status class would be simpler to implement than a full crud API.

**Comment by billz** [comment]
A RESTful API has been implemented thanks in large part to @NL-TCH in https://github.com/RaspAP/raspap-webgui/pull/1519
